.github/workflows/integration-tests.yml

```diff
@@ -8,7 +8,7 @@ env:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
it isn't clear to me why this change?

Reply:
When GitHub updates `ubuntu-latest` to 26.04 next year, the integration tests will start failing again in the same way as [here](https://github.com/mozilla/sccache/pull/2324#issuecomment-2625476046) until the HIP job is updated to `rocm/dev-ubuntu-26.04`.

It will stay broken until a new version is published (or we switch images). This lets us update on our schedule while keeping keep CI green. Since we have to update this file in both cases, I chose the path where jobs don't start mysteriously failing due to GitHub changes.